### PR TITLE
Completed Network Status tests

### LINKED ISSUE #349: Network Status functional tests
Tests written in gherkin, regarding all of the network status tab/section and all of its connected features.

### PR DESCRIPTION
**Pull request by emedrano2112**
Closes #349

### CODE REVIEW

File: func-tests/Network Status/network-information.feature [hunk 1]
```diff
@@ -0,0 +1,2362 @@
...
+        }
+    ]
+    And each {_id}, {ticker}, {networkID}, {network}, {description}, {index}, {blockchain} and {active} matches the information displayed in the UI.
+
+Scenario: Most Popular Chains visual verification
+    Given that the user is in the Pocket Dashboard. Example: 
+    |https://dashboard.testnet.pokt.network/dashboard|
+    When the page is fully loaded.
+    Then make sure to check the Most Popular Chains list, being present.
+    And scroll all the way doen the list.
```
Comment (nymd) [inline func-tests/Network Status/network-information.feature]: "all the way down"

File: func-tests/Network Status/network-information.feature [hunk 2]
```diff
@@ -0,0 +1,2362 @@
...
+    Then user should be able to see the POKT Price. Example: US $0.08
+    And user should be able to see the Total Staked Tokens. Example: 5,000,015,975
+    And user should be able to see the Total Staked Nodes. Example: 5
+    And user should be able to see the Total Staked Apps. Example: 1
+
+Scenario: Network main information verification
+    Given that the user is in the Pocket Dashboard. Example: 
+    |https://dashboard.testnet.pokt.network/dashboard|
+    When the page is fully loaded.
+    And the descripbed main network information is present.
```
Comment (nymd) [inline func-tests/Network Status/network-information.feature]: described

File: func-tests/Network Status/network-information.feature (hunk 1 above)
Comment (nymd) [reply func-tests/Network Status/network-information.feature]: looks like the typo was copypasted so a search/replace should fix them all

**Review by nymd — CHANGES_REQUESTED** [review]
Should it have a section for the "Welcome back USERNAME" alert that is present when you first load the page?

File: func-tests/Network Status/network-information.feature (hunk 2 above)
Comment (emedrano2112) [reply func-tests/Network Status/network-information.feature]: Resolved and thanks!

File: func-tests/Network Status/network-information.feature (hunk 1 above)
Comment (emedrano2112) [reply func-tests/Network Status/network-information.feature]: Resolved and thanks!

**Review by pabelnl — APPROVED** [review]

**Pull request merged**